frontend_tests/node_tests/components.js

```diff
@@ -1,4 +1,5 @@
 set_global('i18n', global.stub_i18n);
+set_global('page_params', {});
 
 zrequire('keydown_util');
 zrequire('components');
```

Comment:
The CI failure is because you need to add tests to this file covering the new corner case.